README.md

```diff
@@ -69,4 +77,10 @@ Please follow the steps below, in order to submit the exercise
 * [Change the local repository `origin` remote URL to your own repository](https://docs.github.com/en/github/using-git/changing-a-remotes-url)
 * [Create a new branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
 * Implement the exercise in your new branch
-* [Create a pull request with your changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) and send it over to us
+* [Create a Pull Request with your changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) and send it over to us
+
+## FAQ
+
+### What programming languages can I use?
+
+You can use the programming language of your choice, although Python and JavaScript are the languages we work with
```

Comment:
Let's not include language suggestions.

Reply:
Resolved in https://github.com/withlogicco/junior-web-dev-exercise/pull/2/commits/4b30f77849489edab7d279ff86b35c157a2974c1.